lib/functions/host/external-toolchains.sh

```diff
@@ -24,9 +24,6 @@ function download_external_toolchains() {
 			# download external Linaro compiler and missing special dependencies since they are needed for certain sources
 
 			local toolchains=(
-				"gcc-linaro-aarch64-none-elf-4.8-2013.11_linux.tar.xz"
-				"gcc-linaro-arm-none-eabi-4.8-2014.04_linux.tar.xz"
-				"gcc-linaro-arm-linux-gnueabihf-4.8-2014.04_linux.tar.xz"
```

Comment:
I don't think this whole function is being used anywhere as SKIP_EXTERNAL_TOOLCHAINS is set to yes by default. Another candidate for removal maybe?

Reply:
Yeah. SKIP_EXTERNAL_TOOLCHAINS=yes was one of the main subjects of armbian-next. it shouldn't be used at all, and the whole code path could be removed, in my opinion.